src/atoms/lp_cone/DotSortAtom.jl

```diff
@@ -32,6 +23,7 @@ function Base.sign(x::DotSortAtom)
     if all(x.w .>= 0)
         return sign(x.children[1])
     elseif all(x.w .<= 0)
+        # FIXME: shouldn't this be -sign?
         return sign(x.children[1])
```

Comment:
@ericphanson just to double-check my understanding, shouldn't this be `-sign(x.children[1])`?

Reply:
This goes right back to the original commit: https://github.com/jump-dev/Convex.jl/commit/4795ded2fcf2a15a0ebf85ab7eb49418d5eb67fd

Reply:
yeah, you are right, looking at the `sign_from_args` part of https://www.cvxpy.org/_modules/cvxpy/atoms/dotsort.html#dotsort.

I never did a thorough sweep of the repo as you're doing, and I'm very glad you're doing it!!

Reply:
Okay, cool. I didn't think to look at the cvxpy implementation.

> I never did a thorough sweep of the repo as you're doing, and I'm very glad you're doing it!!

I have been learning a lot!